### ISSUE TITLE
[Feature] More useful columns in the artists table

### ISSUE THREAD

**Issue by faucetlol2** [Earlopain/reverser#55]
The artists table is a little empty at the moment and could be a good place for some additional statistics.

Mockup:
![image](https://user-images.githubusercontent.com/102884856/189988053-66aca8b1-0415-469b-b8a0-b53baef4dd8c.png)

**Last scraped**: The last scraped date, useful for checking whether it's probably a good time to run the scraper again to check for new posts. _I don't think there's actually any way of checking this without logging into the database at the moment?_

Obviously showing a single value when an artist can have multiple URLs is a little bit problematic. I'm unsure whether it might actually be better to actually show the date on the *least recently* scraped URL to make sure every URL is being kept up to date. A different name than "last scraped" would be required in that case.

**Not uploaded**: The number of results for status "Not Uploaded", minus hidden posts. It would eventually be a personal goal for me to this number down to 0 for each artist.

**Larger**: Ditto for above but the number of submission files that are larger than their e621 counterpart, presumably this would be some sort of combination of `larger_only_filesize_kb`, `larger_only_filesize_percentage` and `larger_only_dimensions`.

**Comment by Earlopain** [comment]
Oh, that sounds nice.

Last scraped is not exposed anywhere right now. All implemented site apis return the creation date of submissions, which means I can use it to stop scraping once I encounter one created before the last scraped date.

I think showing the oldest scraped date on the artist page and _somehow_ displaying the individual pages last scraped date is a good idea. I think it's fine to just keep "Last scraped", no need to waste brain power when it's likely just us two using it anyways. Let's see if I can also somehow add an order option for that. I don't think I have any ordering exposed in the UI yet.

Larger only could be an intersection of the searches with their default treshold. I believe I'll need to add some useful indicies, no way that will be performant lol.

**Comment by faucetlol2** [comment]
Little unrelated to the artists table but while we're on the topic of last scraped dates, it could also be added as title text to each [U] link if we wanted to see it at a more detailed level.

![image](https://user-images.githubusercontent.com/102884856/190010151-d28a9cae-0ae1-40f5-831d-d9751bbdb676.png)

**Comment by Earlopain** [comment]
Added some things. As expected performance tanked, but I think it's still managable.

Load times for me went from ~0.4 to 1.2, or 1.7 for the worst page of all.
![image](https://user-images.githubusercontent.com/14981592/190510633-aed68120-27cc-4c04-b1cd-3546825a7f8a.png)

I did two different columns for `larger_only_filesize_percentage` and `larger_only_dimensions`. It's not feasable to combine the searches to show only the intersection, but this also seems fine.
